Update customizable classes docs (#11207)
RasaHQ/rasa: docs/docs/tracker-stores.mdx

## Custom Tracker Store

If you need a tracker store which is not available out of the box, you can implement your own.
This is done by extending the base class `TrackerStore` and one of the provided mixin classes that implement the
`serialise_tracker` method: `SerializedTrackerAsText` or `SerializedTrackerAsDict`.

To write a custom tracker store, extend the `TrackerStore` base class. Your constructor has to
provide a parameter `host`.
The constructor also needs to make a `super` call to the base class `TrackerStore` using `domain` and `event_broker` arguments:

```
super().__init__(domain, event_broker, **kwargs)
```

> Your custom tracker store class will also have to override three base class methods that require implementation:

Reviewer: ```suggestion
Your custom tracker store class must also implement the following three methods:
```
```suggestion
Your custom tracker store class will also have to override three base class methods that require implementation:
```

- `save`: saves the conversation to the tracker store.
- `retrieve`: retrieves tracker for the latest conversation session.

> - `keys`: returns the set of values for the tracker store's primary key.

Reviewer: Signatures/return values here as well
